query2.php

```diff
@@ -94,7 +94,7 @@
 			<div class = "search_content" style="position: sticky; top: 0vw;">
 				
 				<center><form action="search.php" method="post">
-					<input type="text" autocomplete="off" class="search" style = "background-color: #C2C2C2; font-family: 'Russo One', sans-serif;  font-size: 1vw;" placeholder="Search For Music..." name="search">
+					<input type="text" autocomplete="off" class="search" style = "font-family: 'Roboto', sans-serif; font-size: 1vw;" placeholder="Search For Music..." name="search">

```

Comment:
Majority consists of applying "Roboto" as the heading font and changing the font colour to white from here and below.